src/Cookie.php

```diff
@@ -4,31 +4,23 @@
 
 final class Cookie
 {
-    /** @var string */
-   private $name;
+   private string $name;
```

Comment:
Since package only supports PHP Version >=7.x you can use real typecasts instead of @var tags in comment.
This helps to make the code better readable and better version support.